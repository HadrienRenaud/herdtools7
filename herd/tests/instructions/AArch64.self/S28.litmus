AArch64 S28
{
0:X2=NOP;
0:X1=P1:L0;
}
  P0         | P1           ;
 STR W2,[X1] |L0:           ;
             | B L1         ;
             | ADD W2,W2,#1 ;
             |L1:           ;
             | ADD W2,W2,#2 ;
exists 1:X2=3
