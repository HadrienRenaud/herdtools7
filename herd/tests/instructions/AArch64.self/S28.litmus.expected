Test S28 Allowed
States 2
1:X2=2;
1:X2=3;
Ok
Witnesses
Positive: 1 Negative: 1
Condition exists (1:X2=3)
Observation S28 Sometimes 1 1
Hash=67619acbffb4c487f2a597a5337e9e6c

